packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -399,6 +399,8 @@ class EditableText extends StatefulWidget {
   /// Defaults to EdgeInserts.all(20.0).
   final EdgeInsets scrollPadding;
 
+  static bool debugDeterministicCursor = false;
```

Comment:
Please add a comment here describing what it does and when it is used (only in tests).